openjdk.test.load/config/inventories/mix/mix-more.xml

```diff
@@ -0,0 +1,12 @@
+<!-- 
+  This is a minimal inventory intended for tests which just want to be able
+  to generate a load which can be used for testing other capabilities alongside.
+-->
+
+<inventory>
```

Comment:
Could you please explain which existing Load_Level_4 test this STF load replaces?